### ISSUE TITLE
Rule /require-data-selectors should not warn on cy.get(selector)

### ISSUE THREAD

**Issue by ana-moraes** [cypress-io/eslint-plugin-cypress#270]
Hello,

This issue was discussed before on https://github.com/cypress-io/eslint-plugin-cypress/issues/44, where **_judemorrissey_** reported the warnings being applied on aliases. This part seems to be working fine, but the situation described by **_SeanPercy_** and **_morficus_** is still ongoing.

For example: 
```javascript
// This works
cy.get('[data-test=a]').as('a'); 
cy.get('@a'); 

// This gives a warning
const ASSESSMENT_SUBMIT = '[data-cy=assessment-submit]';
cy.get(ASSESSMENT_SUBMIT).click();

```

This also gives a waning when using separate files for selectors

```javascript
// selectors.js

const addPrefix = testId => `[data-cy=${testId}]`;
export const ASSESSMENT_SUBMIT = addPrefix('assessment-submit');

// test.js
cy.get(ASSESSMENT_SUBMIT).click();
```

Is it possible to change the rule to accommodate this type of situation?

Thank you.
